climada/util/yearsets.py

```diff
@@ -25,31 +25,32 @@
 def impact_yearset(event_impacts, sampled_years=None, sampling_vect=None):
 
     """PURPOSE:
-      Create an annual_impacts object containing a probabilistic impact for each year 
-      in the sampled_years list (or a list generated with the length of given sampled_years) 
+      Create an annual_impacts object containing a probabilistic impact for each year
+      in the sampled_years list (or a list generated with the length of given sampled_years)
       by sampling events from the existing input event_impacts.
 
     INPUTS:
       event_impacts (impact object): impact per event
     OPTIONAL INPUT:
-        sampled_years (int or list): either an integer specifying the amount of years to 
+        sampled_years (int or list): either an integer specifying the amount of years to
             be sampled or a list of years that shall be covered by the resulting annual_impacts
             default: a 1000 year-long list starting in the year 0001
-        sampling_vect (dict): the sampling vector specifying which events contained in the events_impacts
-            are selected in the newly created annual_impacts and the number of events per year in the
-            annual_impacts, therefore the sampling_vect contains two arrays:
+        sampling_vect (dict): the sampling vector specifying which events contained in the
+            events_impacts are selected in the newly created annual_impacts and the number
+            of events per year in the annual_impacts, therefore the sampling_vect contains
+            two arrays:
                 selected_events (array): sampled events (len: total amount of sampled events)
-                events_per_year (array): events per resampled year 
-            The sampling_vector needs to be obtained in a first call, 
+                events_per_year (array): events per resampled year
```

Comment:
Just a thought: with the restructuring and simplification of the module, does the name `resampling` still makes sense, or is `sampling` more appropriate?

Reply:
You're right, I'll change all the "resampling" to "sampling"